### PR TITLE
Update InterventionAdmendment compute_reference_number

### PR DESCRIPTION
**Pull request by reinbach**
Return integer as `number` field is type IntegerField

### CODE REVIEW

File: EquiTrack/partners/models.py [hunk 1]
```diff
@@ -1774,11 +1774,12 @@ class InterventionAmendment(TimeStampedModel):
     tracker = FieldTracker()
 
     def compute_reference_number(self):
+        qs = self.intervention.amendments
         if self.signed_date:
-            return '{0:02d}'.format(self.intervention.amendments.filter(signed_date__isnull=False).count() + 1)
+            seq = qs.filter(signed_date__isnull=False).count() + 1
```
Comment (robertavram) [inline EquiTrack/partners/models.py]: this is an old req... we don't need to save any amendment unless is signed... it used to be that we had to add one to generate the document... now we cannot save un-singed amendments.

Comment (reinbach) [reply EquiTrack/partners/models.py]: So `signed_document` value is expected (it is no longer nullable) when saving the amendement record initially, but `signed_date` value can/is set in a later transaction, possibly? Does the `amendment_number` need to then change/update if/when the `signed_date` value is set in a later transaction?

Comment (robertavram) [reply EquiTrack/partners/models.py]: so.. no.. the amendment number is not changing anymore.. also the signed_date is required by validation... so I believe the else can now be removed.

Comment (reinbach) [reply EquiTrack/partners/models.py]: OK, I've updated the method, by removing the `else` statement

**Pull request merged**